### ISSUE TITLE
Integration Request - Be able to Integrate with AskNicely

### ISSUE THREAD

**Issue by afrahm** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1620]
Have a native integration with AskNicely to get feedback from customers.

**Comment by Uzair-RepairDesk** [comment]
Hey, @afrahm  Hope you are doing well,
I can certainly see how suggested improvements would be useful. With a small team and a big roadmap, we have to pick new features based on the value they’ll add to the most users possible. At the moment, we won’t be able to make this one work.

And I hope that if there’s anything else I can do for you, you won’t hesitate to let me know.

Regards,
Hamza

**Comment by afrahm** [comment]
Yeah that's understandable. But be great to keep this on the radar as more people get interested in it.

**Comment by AMott-2020** [comment]
definitely would love this feature!
